cmd/build/main.go

```diff
@@ -4,15 +4,28 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/convox/convox/pkg/build"
 	"github.com/convox/convox/pkg/structs"
 	"github.com/convox/convox/sdk"
 )
 
+type StringSlice []string
+
+func (i *StringSlice) String() string {
+	return strings.Join(*i, ",")
+}
+
+func (i *StringSlice) Set(value string) error {
```

Comment:
I guess this is needed to satisfy an interface ?

Reply:
yes